media-api/app/controllers/MediaApi.scala

```diff
@@ -120,28 +120,20 @@ class MediaApi(
   }
 
   def getImage(id: String) = auth.async { request =>
-    implicit val r = request
-
-    val include = getIncludedFromParams(request)
-
-    elasticSearch.getImageById(id) map {
-      case Some(source) if hasPermission(request, source) =>
-        val writePermission = canUserWriteMetadata(request, source)
-        val deleteImagePermission = canUserDeleteImage(request, source)
-        val deleteCropsOrUsagePermission = canUserDeleteCropsOrUsages(request.user)
-
-        val (imageData, imageLinks, imageActions) = imageResponse.create(
-          id,
-          source,
-          writePermission,
-          deleteImagePermission,
-          deleteCropsOrUsagePermission,
-          include,
-          request.user.apiKey.tier
-        )
-
+    getImageResponseFromES(id, request) map {
+      case Some((_, imageData, imageLinks, imageActions)) =>
         respond(imageData, imageLinks, imageActions)
+      case _ => ImageNotFound(id)
+    }
+  }
 
+  /**
+    * Get the raw response from ElasticSearch.
+    */
+  def getImageRaw(id: String) = auth.async { request =>
+    getImageResponseFromES(id, request) map {
+      case Some((source, _, imageLinks, imageActions)) =>
+        respond(source, imageLinks, imageActions)
```

Comment:
Do we want the `imageLinks` and `imageActions` on the raw endpoint if it's designed to be an ES proxy? That is, I'm not sure the consumer of this endpoint will be using this data, so not sure its needed.

Reply:
Yes, I added this for consistency with other endpoints -- I considered taking another approach but the Grid docs state 'exposed as hypermedia APIs (argo)'. If you think it's not necessary we can get rid.